GitStartHQ/graphql-engine
# [GQL-11] #8600 Cmd/Ctrl+Enter in the SQL Runner UI should run the SQL statements

### Description


While writing SQL in hasura, Run button has to be clicked for the execution which is a bit clunky. This PR makes the query to execute if Cmd+Enter(Windows: Ctrl+Enter) is hit.

### Changelog


### Affected components



### Related Issues



#8600

### Solution and Design


Now the SQL query can be executed from keyboard with Cmd/Ctrl + Enter.

https://user-images.githubusercontent.com/59878660/176457627-6726193f-c369-4380-89d6-897219573198.mp4

 



### Steps to test and verify


1. Write query in SQL runner
2. Press Ctrl/Cmd + Enter
3. Observe the result of execution of query

### Server checklist


#### Catalog upgrade

Does this PR change Hasura Catalog version?

#### Metadata


Does this PR add a new Metadata feature?
  - Does `run_sql` auto manages the new metadata through schema diffing?
  - Does `run_sql` auto manages the definitions of metadata on renaming?
  - Does `export_metadata`/`replace_metadata` supports the new metadata added?


#### GraphQL
   
   

#### Breaking changes


  1. Metadata API

     Existing `query` types:
     

  4. GraphQL API

     Schema Generation:
     
     
     

     Schema Resolve:-
     
     

  5. Logging
